packages/theming/android-theme/src/colorsTokens.ts

```diff
@@ -199,5 +199,8 @@ export function paletteFromAndroidColors(p: FluentUIAndroidPalette): ThemeColorD
     checkboxBackground: p.buttonBackground,
     checkboxBackgroundDisabled: p.checkboxDisabled,
     checkboxBorderColor: p.checkboxBorder,
+
+    personaActivityRing: p.surfacesPrimary,
+    personaActivityGlow: p.buttonBackground,
```

Comment:
We (the apple / FX team) probably need to add these tokens to the apple theme as well?

Reply:
Yep! Or maybe I could just do it here!

Reply:
Also, where did you get these tokens from?

I ask because I have two feelings:

`personaActivityRing` should just match the PersonaCoin initials background color (which I think is called coinColor), and not really be a token. The same token (something like AvatarBackgroundColor/PersonaBackgroundColor) should change both, with a prop to override the color if you need it something else.

For `personaActivityGlow`, I can't imagine you'd ever want it something other than transparent, or white for light mode / black for dark mode?

Reply:
Okay so `personaActivityRing` refers to the inner and outer rings - the background rings to the main presence ring
And the `personaActivityGlow` refers to the presence ring.
Both these are just default values that have to should be taken if `ringColor` (glow/presence/personaActivityGlow) & `ringBackgroundColor` (personaActivityRing) are not supplied in `RingConfig`.

Reply:
OK I'm looking at what we do in FluentUI Apple Vnext and it looks like we have similar tokens..
https://github.com/microsoft/fluentui-apple/blob/vnext-prototype/ios/FluentUI/Vnext/Avatar/AvatarTokens.swift
https://github.com/microsoft/fluentui-apple/blob/11b9a86f8ef8f2b161604d0ccf8b4908850c3b85/tools/sgen/input/FluentUIStyle.yml#L135

Reply:
I'll ignore this and leave this for a future aligning talk. FYI @EJLayne